disruption_py/workflow.py

```diff
@@ -95,7 +100,8 @@ def get_shots_data(
     Any
         The value of OutputSetting.get_results. See OutputSetting for more details.
     """
-    (log_settings or LogSettings()).setup_logging()
+    log_settings = log_settings or LogSettings()
```

Comment:
could you please add a case where the user just specifies a string (eg "INFO" or "WARNING") to `get_shots_data`, and it's the framework that instantiates a LogSettings object with that default console log level?